frontend/src/metabase/actions/containers/ActionCreator/utils.ts

```diff
@@ -171,7 +172,7 @@ export const convertQuestionToAction = (
     name: question.displayName() as string,
     description: question.description(),
     dataset_query: question.datasetQuery() as NativeDatasetQuery,
-    database_id: question.databaseId(),
+    database_id: question.databaseId() as DatabaseId,
```

Comment:
😭

Reply:
I think it would be better to call `checkNotNull(question.databaseId())` instead of casting a nullable type to a non-nullable one. WDYT?